service/controller/key/zz_generated.ami.go

```diff
@@ -321,6 +325,28 @@ var amiJSON = []byte(`{
     "us-west-1": "ami-006da855103391bd8",
     "us-west-2": "ami-0bef8aecb982525a5"
   },
+  "2605.12.0": {
+    "ap-east-1": "ami-0e61d15ce4cd16733",
+    "ap-northeast-1": "ami-06471d461a6d5985b",
+    "ap-northeast-2": "ami-0a992ed1dbf215084",
```

Comment:
Why do we have these changes in here now?

Reply:
I accidentally pushed changes into the wrong branch :sob:  I will revert for now